treeherder/autoclassify/autoclassify.py

```diff
@@ -3,13 +3,18 @@
 
 from django.db.utils import IntegrityError
 
-from treeherder.model.models import (FailureLine,
+from treeherder.model.models import (ClassifiedFailure,
                                      FailureMatch,
+                                     Job,
                                      JobNote,
                                      Matcher,
+                                     TextLogError,
                                      TextLogErrorMatch)
 
 logger = logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
```

Comment:
I don't think we want to set the DEBUG level here - settings.py's `LOGGING` dict seems like the correct place? (And I'm curious why we'd need to override the defaults there?)